pycbc/workflow/core.py

```diff
@@ -197,18 +197,26 @@ def __init__(self, cp, name,
 
         if exe_url.scheme in ['', 'file']:
             if os.path.isfile(exe_url.path):
-                self.add_pfn(exe_path)
+                self.add_pfn(exe_path,site='local')
 
                 logging.debug("Using %s executable "
                               "at %s" % (name, exe_url.path))
             else:
                 raise TypeError("Failed to find %s executable " 
                             "at %s" % (name, exe_path))
         else:
-            # Could be http, gsiftp, etc.  Let Pegasus handle it.
+            # Could be http, gsiftp, etc.
             logging.debug("Using %s executable "
                           "at %s" % (name, exe_path))
-            self.add_pfn(exe_path, site='nonlocal')
+            try:
+                value = string.strip(
```

Comment:
The first strip here is not needed. Configparse strips all values when reading an ini file.